OrbitCore/Utils.h

```diff
@@ -110,18 +112,23 @@ inline bool IsAllDigits(const std::string_view value) {
   return std::all_of(value.begin(), value.end(), IsDigit);
 }
 
+namespace OrbitCore {
+namespace OrbitUtils {
+
 //-----------------------------------------------------------------------------
-inline std::string FileToString(const std::string_view a_FileName) {
-  std::string file_name(a_FileName);
-  std::stringstream buffer;
-  std::ifstream inFile(file_name);
-  if (!inFile.fail()) {
-    buffer << inFile.rdbuf();
-    inFile.close();
+inline outcome::result<std::string> FileToString(const std::string& file_name) {
+  std::ifstream file_stream(file_name);
+  if (file_stream.fail()) {
+    return outcome::failure(OrbitCore::Error::kFailedToReadFile);
```

Comment:
I think `outcome::failure(static_cast<std::errc>(errno))` should work as well for iostream-stuff.

Reply:
This basically then removes the need for the dedicated Error class. Will remove.

Reply:
At least for these two use cases, yes. There might be others where we will need it again.

Heads up: I'm not 100% sure if errno works with iostream. It would be great if you could at least try this once. :see_no_evil:

Reply:
my research says yes, but i will try it.